constants/index.js

```diff
@@ -0,0 +1,32 @@
+const PATH_AVATARS = "/public/avatars";
+const AVATAR_GUEST = "guest";
+
+const DEFAULT_USER = "system";
+
+const SALT_ROUNDS = 12;
+const RANGE = 1000;
+const INVALID_EMAIL_ADDRESS = "Invalid Email Address!";
```

Comment:
What is this `RANGE` indicating? could you change the name for something more meaningful?

Reply:
It's for generating a pepper for the user's password. Would changing it to PEPPER_RANGE suffice?

Reply:
Yes